src/robot/output/pyloggingconf.py

```diff
@@ -56,6 +56,9 @@ def set_level(level):
 
 
 class RobotHandler(logging.Handler):
+    def __init__(self, level=logging.NOTSET, library_logger=librarylogger):
```

Comment:
Our convention has been adding an empty row above `__init__`. I know Black doesn't do that but arguable having it is in line with PEP-8. I don't care super strongly about that, but I do care about consistency and would prefer that empty line here. Preferably they should be added to tests as well.